server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -40,10 +40,12 @@
 import org.kaazing.gateway.util.Utils;
 import org.kaazing.gateway.util.ssl.SslCipherSuites;
 import org.kaazing.gateway.util.ws.WebSocketWireProtocol;
+import static org.kaazing.gateway.service.TransportOptionNames.*;
 
 public class DefaultConnectOptionsContext implements ConnectOptionsContext {
 
     private static final long DEFAULT_WS_INACTIVITY_TIMEOUT_MILLIS = 0L;
+    private static int DEFAULT_HTTP_KEEPALIVE_TIMEOUT = 30; //seconds
```

Comment:
Are we changing the externally perceived behavior of the product by altering the default value of one of the connect options?